### PR TITLE
(DOCSP-21146): Refactor to improve readability from C to B or grade 15 to grade 8

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCSP-21146

### Staged Changes (Requires MongoDB Corp SSO)

- [Authentication Providers](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-21146/authentication/providers/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/authentication/providers.txt [hunk 1]
```diff
@@ -128,62 +122,40 @@ object:
      - Details
    
    * - :doc:`Anonymous </authentication/anonymous>`
-     - Anonymous users are not associated with any metadata.
+     - Anonymous users do not have any metadata.
```
Comment (nathan-contino-mongo) [inline source/authentication/providers.txt:125]:
```suggestion
     - Anonymous users have no metadata.
```
Even simpler?

File: source/authentication/providers.txt [hunk 2]
```diff
@@ -19,44 +19,38 @@ Authentication Providers
...
-account from one provider to a user account from another
-provider by using the client SDKs.
-
-An example of an app that would benefit from multiple
-authentication providers is a blog or news service. The
-typical user of such an app would authenticate anonymously
-so that they don't need to register. However, the blog
-authors or journalists would need to sign in with some other
-provider to be authorized to publish new content.
+Authentication providers log users in to your client application.
```
Comment (nathan-contino-mongo) [inline source/authentication/providers.txt:22]: Something about this sentence is really hard for me to parse -- I had to read it 2 or 3 times to understand what it was saying. Maybe I just have "moving across the country tomorrow" brain, but I wonder if we could rephrase to something like this:

"Users log in to your client application using authentication providers."

This is a really subjective take though so if you feel the current sentence is easier to understand, stick with it.

**Review by nathan-contino-mongo — APPROVED** [review]
LGTM

File: source/authentication/providers.txt (hunk 2 above)
Comment (dacharyc) [reply source/authentication/providers.txt:22]: No, I agree this felt a little weird after updating it. I like your version! Running with it.

**Pull request merged**